src/sparseml/exporters/kv_cache_injector.py

```diff
@@ -71,25 +78,31 @@ def __init__(
         self.inplace = inplace
         self.config = self.get_config(model_path)
 
-        if not self.config["model_type"] in _SUPPORTED_ARCHITECTURES:
+        if self.config["model_type"] not in PositionEmbeddingTransformation.__members__:
```

Comment:
as mentioned in #1573 let's enable some default behavior here

Reply:
Solved in a different PR